mathics/builtin/list/eol.py

```diff
@@ -589,24 +590,24 @@ class First(Builtin):
     >> First[a + b + c]
      = a
     >> First[x]
-     : Nonatomic expression expected.
+     : Nonatomic expression expected at position 1 in First[x].
      = First[x]
     >> First[{}]
      : {} has zero length and no first element.
      = First[{}]
     """
 
```

Comment:
`First` and `Last`should have a default value as second arg and attribute `A_HOLD_REST`.

Reply:
```
In[1]:= First[a]                                                                

First::normal: Nonatomic expression expected at position 1 in First[a].

Out[1]= First[a]

In[2]:= First[a, b]                                                             

Out[2]= b

In[3]:= First[a, b, c]                                                          

First::argt: First called with 3 arguments; 1 or 2 arguments are expected.

Out[3]= First[a, b, c]

```

Reply:
Thanks -  this should be handled now.